Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3109 @@
+{
```

Comment:
<p>I think doing <code>adults['A'] = (adults['age'] - 18) / (65 - 18)</code> or <code>adults['A'] = (adults['age'].copy() - 18) / (65 - 18)</code> will get rid if the warning (I don't remember exactly :D)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>